test/e2e_playwright/tests/surveyInviteUser.js

```diff
@@ -0,0 +1,37 @@
+import { DataTestId, getSelector } from '../../../webapp/utils/dataTestId'
+import { gotoSurveyUsersList } from './_navigation'
+
+const expectUser = async ({ user }) => {
+  await expect(page).toHaveText(user)
```

Comment:
you can access the specific row and then check the right column
```
const emailEl = await page.$('div[data-testid="users-${0}"] div[data-testid="user-email"]')
expect(await emailEl.getInnerText()).toBe(user))
```

Reply:
changed